<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1003</width>
    <height>670</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>991</width>
      <height>561</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Setting</string>
     </attribute>
     <widget class="QPushButton" name="testtradingbtn">
      <property name="geometry">
       <rect>
        <x>310</x>
        <y>460</y>
        <width>191</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>테스트 거래 시작</string>
      </property>
     </widget>
     <widget class="QPushButton" name="livetradingstartbtn">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>460</y>
        <width>191</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>실시간 거래 시작</string>
      </property>
     </widget>
     <widget class="QTabWidget" name="tabWidget_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>971</width>
        <height>441</height>
       </rect>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab_5">
       <attribute name="title">
        <string>Strategy</string>
       </attribute>
       <widget class="QGroupBox" name="groupBox">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>10</y>
          <width>481</width>
          <height>401</height>
         </rect>
        </property>
        <property name="title">
         <string/>
        </property>
        <widget class="QLabel" name="label">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>10</y>
           <width>64</width>
           <height>15</height>
          </rect>
         </property>
         <property name="text">
          <string>전략</string>
         </property>
        </widget>
        <widget class="QComboBox" name="strategylistcombo">
         <property name="geometry">
          <rect>
           <x>70</x>
           <y>10</y>
           <width>161</width>
           <height>22</height>
          </rect>
         </property>
        </widget>
        <widget class="QComboBox" name="minutecombo">
         <property name="geometry">
          <rect>
           <x>70</x>
           <y>50</y>
           <width>131</width>
           <height>22</height>
          </rect>
         </property>
        </widget>
        <widget class="QLabel" name="label_2">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>50</y>
           <width>64</width>
           <height>15</height>
          </rect>
         </property>
         <property name="text">
          <string>분봉</string>
         </property>
        </widget>
        <widget class="QPushButton" name="strategyaddbtn">
         <property name="geometry">
          <rect>
           <x>40</x>
           <y>360</y>
           <width>93</width>
           <height>28</height>
          </rect>
         </property>
         <property name="text">
          <string>전략 추가</string>
         </property>
        </widget>
        <widget class="QPushButton" name="strategydelbtn">
         <property name="geometry">
          <rect>
           <x>150</x>
           <y>360</y>
           <width>93</width>
           <height>28</height>
          </rect>
         </property>
         <property name="text">
          <string>전략 제거</string>
         </property>
        </widget>
        <widget class="QPushButton" name="checkstrategybtn">
         <property name="geometry">
          <rect>
           <x>270</x>
           <y>360</y>
           <width>93</width>
           <height>28</height>
          </rect>
         </property>
         <property name="text">
          <string>전략 확인</string>
         </property>
        </widget>
       </widget>
       <widget class="QListWidget" name="strategylist">
        <property name="geometry">
         <rect>
          <x>625</x>
          <y>10</y>
          <width>331</width>
          <height>401</height>
         </rect>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_6">
       <attribute name="title">
        <string>Trader</string>
       </attribute>
      </widget>
      <widget class="QWidget" name="tab_7">
       <attribute name="title">
        <string>Log</string>
       </attribute>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>View</string>
     </attribute>
     <widget class="QLabel" name="statuslabel">
      <property name="geometry">
       <rect>
        <x>480</x>
        <y>490</y>
        <width>491</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="chartbtn">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>480</y>
        <width>191</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>차트버튼</string>
      </property>
     </widget>
     <widget class="QTabWidget" name="chartTabwidget">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>971</width>
        <height>421</height>
       </rect>
      </property>
      <property name="currentIndex">
       <number>2</number>
      </property>
      <widget class="QWidget" name="tab_3">
       <attribute name="title">
        <string>1min</string>
       </attribute>
       <widget class="QWidget" name="chartWidget1" native="true">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>961</width>
          <height>391</height>
         </rect>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_4">
       <attribute name="title">
        <string>5min</string>
       </attribute>
       <widget class="QWidget" name="chartWidget2" native="true">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>961</width>
          <height>391</height>
         </rect>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_9">
       <attribute name="title">
        <string>30min</string>
       </attribute>
       <widget class="QWidget" name="chartWidget4" native="true">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>961</width>
          <height>391</height>
         </rect>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_8">
       <attribute name="title">
        <string>1hour</string>
       </attribute>
       <widget class="QWidget" name="chartWidget3" native="true">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>961</width>
          <height>391</height>
         </rect>
        </property>
       </widget>
      </widget>
     </widget>
     <widget class="QPushButton" name="checkdatabtn">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>480</y>
        <width>191</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>데이터 확인 버튼</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1003</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
